app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -486,11 +491,57 @@ class MainActivity : BaseActivity() {
         syncAutoConfigActivityResultLauncher.launch(Intent(this, SyncAutoConfigActivity::class.java))
     }
 
+    fun easterEggConfettiHasBeenClicked(from: String) {
+
+        val currentTime = System.currentTimeMillis()
+
+        if (easterEggConfettiTime == 0L || currentTime - easterEggConfettiTime > EASTER_EGG_CONFETTI_DELAY_TRIGGER) {
+            easterEggConfettiTime = currentTime
+            easterEggConfettiCount = 1
+        } else {
+            easterEggConfettiCount++
+        }
+
+        if (easterEggConfettiCount == EASTER_EGG_CONFETTI_TAPS_TRIGGER) {
+            mainViewModel.easterEggConfettiTrigger.value = from
+            easterEggConfettiCount = 0
+        }
+    }
+
+    private fun observeEasterEggConfettiTriggers() = with(mainViewModel) {
+        easterEggConfettiTrigger.observeNotNull(owner = this@MainActivity) { from ->
+
+            Sentry.withScope { scope ->
+                scope.level = SentryLevel.INFO
+                scope.setTag("from", from)
+                Sentry.captureMessage("Easter egg Confetti has been triggered! Woohoo!")
+            }
```

Comment:
I think we want to have this kind of info on matomo rather than on sentry

Reply:
I strongly support this